tests/rest_scripts/calculate_checksum.sh

```diff
@@ -16,29 +16,64 @@
 
 source ./tests/rest_scripts/rest.sh
 
-if ! DEACTIVATE=false source ./tests/rest_scripts/init_python_env.sh; then
-  log_rest 2 "error initializing python environment"
-  exit 1
-fi
-if ! checksum_decimal=$(python3 -c "
+calculate_checksum_python() {
+  if [ "$#" -ne 2 ]; then
+    log 2 "'calculate_checksum_python' requires checksum type, data file"
+    return 1
+  fi
+  if ! DEACTIVATE=false source ./tests/rest_scripts/init_python_env.sh; then
+    log_rest 2 "error initializing python environment"
+    return 1
+  fi
+  if ! checksum_decimal=$(python3 -c "
```

Comment:
Having this script have its own inline subscript seems a little janky.  Maybe have the python script be its own thing that is called out to?